### DISCUSSION TITLE
how to: run makefile in repo to build app?

### DISCUSSION THREAD

**Discussion by mieliespoor** [goreleaser/goreleaser#3530] [Q&A] [answered]
We have a Makefile in the repo that handles the versioning and compiling of the app. How do we use goreleaser to use this make file? Alternatively, how to I customise the commands to calculate the version number and compile commands?

**Comment by caarlos0** [comment, marked as answer]
you can't use your makefile targets inside goreleaser, although you can make your targets call `goreleaser build` and etc.
